python/dgl/contrib/cugraph/nn/conv/entity_sample.py

```diff
@@ -0,0 +1,133 @@
+import torch
```

Comment:
We probably need to move this to the examples directory. 

https://github.com/dmlc/dgl/tree/master/examples